skimage/measure/block.py

```diff
@@ -17,11 +17,11 @@ def block_reduce(image, block_size, func=np.sum, cval=0, **func_kwargs):
         Function object which is used to calculate the return value for each
         local block. This function must implement an ``axis`` parameter.
         Primary functions are ``numpy.sum``, ``numpy.min``, ``numpy.max``,
-        ``numpy.mean`` and ``numpy.median``.
+        ``numpy.mean`` and ``numpy.median``.  See also `func_kwargs`.
     cval : float
         Constant padding value if image is not perfectly divisible by the
         block size.
-    **func_kwargs :
+    func_kwargs : dict
         Keyword arguments passed to `func`. Notably useful for passing dtype
         argument to ``np.mean``.
```

Comment:
I think this should be:

Keyword arguments passed to `func`. Notably useful for passing dtype
argument to ``np.mean``. Takes dictionaries of inputs, i.e.:
``func_kwargs={'dtype': np.float16})``